src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -763,28 +763,38 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
             shouldShow: shouldShowCategories,
             isSupplementary: !isCategoryRequired,
         },
-        ..._.map(policyTagLists, ({name}, index) => ({
-            item: (
-                <MenuItemWithTopDescription
-                    key={name}
-                    shouldShowRightIcon={!isReadOnly}
-                    title={TransactionUtils.getTagForDisplay(transaction, index)}
-                    description={name}
-                    numberOfLinesTitle={2}
-                    onPress={() =>
-                        Navigation.navigate(
-                            ROUTES.MONEY_REQUEST_STEP_TAG.getRoute(CONST.IOU.ACTION.CREATE, iouType, index, transaction.transactionID, reportID, Navigation.getActiveRouteWithoutParams()),
-                        )
-                    }
-                    style={[styles.moneyRequestMenuItem]}
-                    disabled={didConfirm}
-                    interactive={!isReadOnly}
-                    rightLabel={isTagRequired ? translate('common.required') : ''}
-                />
-            ),
-            shouldShow: shouldShowTags,
-            isSupplementary: !isTagRequired,
-        })),
+        ..._.map(policyTagLists, ({name, required}, index) => {
+            const isTagRequired = !isUndefined(required) && required;
```

Comment:
Why we removed `canUseViolations`?
And this does not work for single level tags where required is not set for each tag, right?

Reply:
Is it still on HOLD? What PR?